#,Node,Feature,TestCase,Result,ResultType,ExecutedOn,Description,Function,Module,Devices,KnownIssue
1,gw3,Regression,ft_add_unknownvlan_interface,Pass,Mapped,2023-12-25 01:05:32,Test case passed,test_ft_add_unknownvlan_interface,switching/test_vlan.py,"V1, V2",
2,gw3,Regression,ft_vlan_delete_with_member,Pass,Mapped,2023-12-25 01:05:46,Test case passed,test_ft_vlan_delete_with_member,switching/test_vlan.py,"V1, V2",
3,gw2,Regression,FtOpSoSwLagFn042,Pass,Mapped,2023-12-25 01:06:15,Test case passed,test_ft_portchannel_with_vlan_variations,switching/test_portchannel.py,"V3, V4",
4,gw2,Regression,FtOpSoSwLagFn041,Pass,Mapped,2023-12-25 01:06:15,Test case passed,test_ft_portchannel_with_vlan_variations,switching/test_portchannel.py,"V3, V4",
5,gw2,PortChannel Config Enhancements,FtOpSoSwPominFn007,Pass,Mapped,2023-12-25 01:06:39,Test case passed,test_ft_verify_min_links_functionality_by_flapping_all_member_ports_007,switching/test_portchannel.py,"V3, V4",
6,gw3,Regression,ft_vlan_trunk_tagged,Pass,Mapped,2023-12-25 01:07:13,Test case passed,test_ft_vlan_trunk_tagged,switching/test_vlan.py,"V1, V2",
7,gw3,Regression,ft_show_vlan_brief,Pass,Mapped,2023-12-25 01:07:13,Test case passed,test_ft_vlan_trunk_tagged,switching/test_vlan.py,"V1, V2",
8,gw2,Regression,FtOpSoSyInIpFn002,DUTFail,Mapped,2023-12-25 01:14:03,Error response from daemon: Container is not running,test_ft_port_fn_verify_shut_noshut,system/test_interface.py,"V3, V4",
9,gw3,warmboot,ft_reboot_fdb_fast_reboot,ScriptError,Mapped,2023-12-25 01:19:22,Exception TypeError 'NoneType' object is not subscriptable in file /apis/system/basic.py at line 1389,test_ft_vlan_save_config_warm_and_fast_reboot,switching/test_vlan.py,"V1, V2",
10,gw3,warmboot,ft_max_vlan_save_reload,ScriptError,Mapped,2023-12-25 01:19:22,Exception TypeError 'NoneType' object is not subscriptable in file /apis/system/basic.py at line 1389,test_ft_vlan_save_config_warm_and_fast_reboot,switching/test_vlan.py,"V1, V2",
11,gw3,warmboot,FtOpSoSysFRFn005,ScriptError,Mapped,2023-12-25 01:19:22,Exception TypeError 'NoneType' object is not subscriptable in file /apis/system/basic.py at line 1389,test_ft_vlan_save_config_warm_and_fast_reboot,switching/test_vlan.py,"V1, V2",
12,gw2,Regression,ft_lldp_LocManAddrLen,Fail,Mapped,2023-12-25 01:21:56,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrLen,system/test_lldp.py,"V3, V4",
13,gw2,Regression,ft_lldp_LocManAddrlfld,Fail,Mapped,2023-12-25 01:21:56,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrlfld,system/test_lldp.py,"V3, V4",
14,gw2,Regression,ft_lldp_LocManAddrEntry,Fail,Mapped,2023-12-25 01:21:56,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrEntry,system/test_lldp.py,"V3, V4",
15,gw2,Regression,ft_lldp_ConfigManAddrEntry,Fail,Mapped,2023-12-25 01:21:57,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_ConfigManAddrEntry,system/test_lldp.py,"V3, V4",
16,gw2,Regression,ft_lldp_lldplocportid,Fail,Mapped,2023-12-25 01:21:57,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocportid,system/test_lldp.py,"V3, V4",
17,gw2,Regression,ft_lldp_lldplocsysname,Fail,Mapped,2023-12-25 01:21:58,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocsysname,system/test_lldp.py,"V3, V4",
18,gw2,Regression,ft_lldp_lldplocsysdesc,Fail,Mapped,2023-12-25 01:21:58,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocsysdesc,system/test_lldp.py,"V3, V4",
19,gw2,SNMP Agent Optimization,ft_snmp_lldp_prem_manaddr_table,Fail,Mapped,2023-12-25 01:21:59,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_rem_man_addr_table,system/test_lldp.py,"V3, V4",
20,gw2,Regression,FtOpSoSwlldpFn011,Fail,Mapped,2023-12-25 01:21:59,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
21,gw2,Regression,FtOpSoSwlldpFn010,Fail,Mapped,2023-12-25 01:21:59,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
22,gw2,Regression,FtOpSoSwlldpFn009,Fail,Mapped,2023-12-25 01:21:59,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
23,gw2,Regression,FtOpSoSwlldpFn008,Fail,Mapped,2023-12-25 01:21:59,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
24,gw2,Regression,FtOpSoSwlldpFn004,Fail,Mapped,2023-12-25 01:21:59,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
25,gw2,Regression,FtOpSoSwlldpFn013,Fail,Mapped,2023-12-25 01:21:59,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_docker_restart,system/test_lldp.py,"V3, V4",
26,gw2,Regression,lldp_verify_counters,Fail,Mapped,2023-12-25 01:22:00,module configuration failed. LLDP neighbors information is not found after polling @5445,test_lldp_verify_counters,system/test_lldp.py,"V3, V4",
27,gw2,Regression,ft_lldp_with_remote_server,Fail,Mapped,2023-12-25 01:22:00,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_with_remote_server,system/test_lldp.py,"V3, V4",
28,gw3,Regression,ft_ssh_service_disable,Pass,Mapped,2023-12-25 01:25:47,Test case passed,test_ft_ssh_service_disable,system/test_ssh.py,"V1, V2",
29,gw5,Instrumentation,ft_sf_wm_telemetry_interval,Pass,Mapped,2023-12-25 01:29:36,verification of configured telemetry interval and reset to default interval is successful,test_ft_watermark_telemetry_interval,system/test_snapshot.py,V3,
30,gw5,Instrumentation Enhancements,ft_sf_snapshot_tam_Error_Logs,Fail,SubTest,2023-12-25 01:29:36,Error logs are getting generated on the device,test_ft_watermark_telemetry_interval,system/test_snapshot.py,V3,
31,gw4,Regression,ft_snmp_sysName,Fail,Mapped,2023-12-25 01:30:18,sysname output is not in sync between system and snmp,test_ft_snmp_sysName,system/test_snmp.py,V4,
32,gw4,Regression,ft_snmp_test_syUpTime,Pass,Mapped,2023-12-25 01:30:24,Test case passed,test_ft_snmp_test_syUpTime,system/test_snmp.py,V4,
33,gw4,Regression,ft_snmp_sysDescr,Pass,Mapped,2023-12-25 01:30:32,Test case passed,test_ft_snmp_sysDescr,system/test_snmp.py,V4,
34,gw4,Regression,FtOpSoSySnFn008,Pass,Mapped,2023-12-25 01:30:39,Test case passed,test_ft_snmp_mib_2,system/test_snmp.py,V4,
35,gw4,Regression,FtOpSoSySnFn009,Pass,Mapped,2023-12-25 01:30:44,Test case passed,test_ft_snmp_if_mib_all,system/test_snmp.py,V4,
36,gw4,Regression,FtOpSoSySnFn010,Pass,Mapped,2023-12-25 01:30:48,Test case passed,test_ft_snmp_entity_mib_all,system/test_snmp.py,V4,
37,gw4,Regression,FtOpSoSySnFn013,Pass,Mapped,2023-12-25 01:30:53,Test case passed,test_ft_snmp_dot1q_dot1db_mib,system/test_snmp.py,V4,
38,gw4,Regression,FtOpSoSySnFn012,Pass,Mapped,2023-12-25 01:30:53,Test case passed,test_ft_snmp_dot1q_dot1db_mib,system/test_snmp.py,V4,
39,gw4,Regression,FtOpSoSySnFn015,Pass,Mapped,2023-12-25 01:31:02,Test case passed,test_ft_snmp_root_node_walk,system/test_snmp.py,V4,
40,gw4,Regression,FtOpSoSySnFn016,Pass,Mapped,2023-12-25 01:31:13,Test case passed,test_ft_snmp_ipAddressRowStatus_ipv6,system/test_snmp.py,V4,
41,gw4,Regression,FtOpSoSySnFn017,Pass,Mapped,2023-12-25 01:31:18,Test case passed,test_ft_snmp_ipAddressStorageType_ipv6,system/test_snmp.py,V4,
42,gw4,Regression,FtOpSoSySnFn020,Pass,Mapped,2023-12-25 01:31:23,Test case passed,test_ft_snmp_ipv6_If_Forward_default_HopLimit,system/test_snmp.py,V4,
43,gw4,Regression,FtOpSoSySnFn021,Pass,Mapped,2023-12-25 01:31:28,Test case passed,test_ft_snmp_ipv6scope_index_table,system/test_snmp.py,V4,
44,gw4,Regression,test_snmp_ip_forward_table,Pass,Mapped,2023-12-25 01:31:36,Test case passed,test_ft_snmp_ip_forward_table,system/test_snmp.py,V4,
45,gw4,SNMP Agent Optimization,ft_snmp_ifXTable,Pass,Mapped,2023-12-25 01:31:42,Test case passed,test_ft_snmp_ifx_table,system/test_snmp.py,V4,
46,gw4,SNMP Agent Optimization,ft_snmp_ip_System_Stats_Table,Pass,Mapped,2023-12-25 01:31:47,Test case passed,test_ft_snmp_ip_System_Stats_Table,system/test_snmp.py,V4,
47,gw4,SNMP Agent Optimization,ft_snmp_ip_IfStats_Table,Pass,Mapped,2023-12-25 01:31:52,Test case passed,test_ft_snmp_ip_IfStats_Table,system/test_snmp.py,V4,
48,gw4,SNMP Agent Optimization,ft_snmp_ip_Address_Table,Pass,Mapped,2023-12-25 01:31:57,Test case passed,test_ft_snmp_ip_Address_Table,system/test_snmp.py,V4,
49,gw4,SNMP Agent Optimization,ft_snmp_ip_NetToPhysical_Table,Pass,Mapped,2023-12-25 01:32:02,Test case passed,test_ft_snmp_ip_NetToPhysical_Table,system/test_snmp.py,V4,
50,gw4,SNMP Agent Optimization,ft_snmp_icmp_Msgs,Pass,Mapped,2023-12-25 01:32:07,Test case passed,test_ft_snmp_icmp_Msgs,system/test_snmp.py,V4,
51,gw4,SNMP Agent Optimization,ft_snmp_tcp_mib,Pass,Mapped,2023-12-25 01:32:14,Test case passed,test_ft_snmp_tcp_mib,system/test_snmp.py,V4,
52,gw4,SNMP Agent Optimization,ft_snmp_udp_mib,Pass,Mapped,2023-12-25 01:32:19,Test case passed,test_ft_snmp_udp_mib,system/test_snmp.py,V4,
53,gw4,SNMP Agent Optimization,ft_snmp_snmpv2_mib,Pass,Mapped,2023-12-25 01:32:24,Test case passed,test_ft_snmp_snmpv2_mib,system/test_snmp.py,V4,
54,gw4,SNMP Agent Optimization,ft_snmp_host_resource_mib,Pass,Mapped,2023-12-25 01:32:29,Test case passed,test_ft_snmp_host_resource_mib,system/test_snmp.py,V4,
55,gw4,SNMP Agent Optimization,ft_snmp_framework_mib,Pass,Mapped,2023-12-25 01:32:34,Test case passed,test_ft_snmp_framework_mib,system/test_snmp.py,V4,
56,gw4,SNMP Agent Optimization,ft_snmp_mpd_mib,Pass,Mapped,2023-12-25 01:32:39,Test case passed,test_ft_snmp_mpd_mib,system/test_snmp.py,V4,
57,gw4,SNMP Agent Optimization,ft_snmp_target_mib,Pass,Mapped,2023-12-25 01:32:44,Test case passed,test_ft_snmp_target_mib,system/test_snmp.py,V4,
58,gw4,SNMP Agent Optimization,ft_snmp_notification_mib,Pass,Mapped,2023-12-25 01:32:49,Test case passed,test_ft_snmp_notification_mib,system/test_snmp.py,V4,
59,gw4,SNMP Agent Optimization,ft_snmp_user_based_sm_mib,Pass,Mapped,2023-12-25 01:32:53,Test case passed,test_ft_snmp_user_based_sm_mib,system/test_snmp.py,V4,
60,gw4,SNMP Agent Optimization,ft_snmp_view_based_acm_mib,Pass,Mapped,2023-12-25 01:32:58,Test case passed,test_ft_snmp_view_based_acm_mib,system/test_snmp.py,V4,
61,gw4,SNMP Agent Optimization,ft_snmp_ent_physical_table,Pass,Mapped,2023-12-25 01:33:03,Test case passed,test_ft_snmp_ent_physical_table,system/test_snmp.py,V4,
62,gw4,SNMP Agent Optimization,ft_snmp_dot3_stats_table,Pass,Mapped,2023-12-25 01:33:08,Test case passed,test_ft_snmp_dot3_stats_table,system/test_snmp.py,V4,
63,gw4,SNMP Agent Optimization,ft_net_snmp_agent_mib,Pass,Mapped,2023-12-25 01:33:13,Test case passed,test_ft_net_snmp_agent_mib,system/test_snmp.py,V4,
64,gw4,SNMP Agent Optimization,ft_net_snmp_vacm_mib,Pass,Mapped,2023-12-25 01:33:18,Test case passed,test_ft_net_snmp_vacm_mib,system/test_snmp.py,V4,
65,gw4,SNMP Agent Optimization,ft_snmp_ucd_diskio_mib,Pass,Mapped,2023-12-25 01:33:23,Test case passed,test_ft_snmp_ucd_diskio_mib,system/test_snmp.py,V4,
66,gw4,SNMP Agent Optimization,ft_snmp_ucd_la_table,Pass,Mapped,2023-12-25 01:33:28,Test case passed,test_ft_snmp_ucd_la_table,system/test_snmp.py,V4,
67,gw4,SNMP Agent Optimization,ft_snmp_ucd_system_stats,Pass,Mapped,2023-12-25 01:33:33,Test case passed,test_ft_snmp_ucd_system_stats,system/test_snmp.py,V4,
68,gw4,SNMP MIB and Traps,SNMPQBr003,Fail,Mapped,2023-12-25 01:33:39,SNMP output is not found after walk of dot1qTpFdbPort OID,test_ft_snmp_dot1q_tp_fdb_port,system/test_snmp.py,V4,
69,gw4,SNMP MIB and Traps,SNMPQBr002,Pass,Mapped,2023-12-25 01:33:45,Test case passed,test_ft_snmp_dot1q_tp_fdb_address,system/test_snmp.py,V4,
70,gw4,In-Memory Debug Log,FtOpSoSysImLogFn012,EnvFail,Mapped,2023-12-25 01:34:04,Test case Failed Couldn't get the snmptrapd process details from snmp trap server. Please check the snmp trap server process in the server.,test_ft_snmp_coldstart_trap,system/test_snmp.py,V4,
71,gw4,SNMP MIB and Traps,SNMPTrap004,EnvFail,Mapped,2023-12-25 01:34:04,Test case Failed Couldn't get the snmptrapd process details from snmp trap server. Please check the snmp trap server process in the server.,test_ft_snmp_coldstart_trap,system/test_snmp.py,V4,
72,gw4,Regression,FtOpSoSySnFn023,Fail,Mapped,2023-12-25 01:34:22,sysname output is not in sync between system and snmp after docker restart,test_ft_snmp_docker_restart,system/test_snmp.py,V4,
73,gw3,Regression,ft_ssh_add_user_verify,Fail,Mapped,2023-12-25 01:36:15,Test case Failed.,test_ft_ssh_add_user_verify,system/test_ssh.py,"V1, V2",
74,gw3,Control Plane ACL,test_ft_controlplane_acl_service_sshv4,Pass,SubTest,2023-12-25 01:36:15,SSH failed,test_ft_ssh_add_user_verify,system/test_ssh.py,"V1, V2",
75,gw3,Control Plane ACL,test_ft_controlplane_acl_service_sshv6,Pass,SubTest,2023-12-25 01:36:15,SSH failed,test_ft_ssh_add_user_verify,system/test_ssh.py,"V1, V2",
76,gw3,Control Plane ACL,test_ft_controlplane_acl_service_snmp,Fail,SubTest,2023-12-25 01:36:15,SNMP output is not found after walk of with control plane ACL service SNMP after reboot OID,test_ft_ssh_add_user_verify,system/test_ssh.py,"V1, V2",
77,gw3,Regression,CETA_SONIC_67575,Pass,Mapped,2023-12-25 01:36:40,Test case passed,test_ft_verify_logs_at_the_time_of_ssh_login_success,system/test_ssh.py,"V1, V2",
78,gw5,Regression,ft_ntp_message_log_display_with_correct_time,ScriptError,Mapped,2023-12-25 01:40:36,Exception IndexError list index out of range in file /apis/system/ntp.py at line 347,test_ft_ntp_disable_enable_with_message_log,system/test_ntp.py,V3,
79,gw5,Regression,ft_ntp_disable_ntp_enable_ntp,ScriptError,Mapped,2023-12-25 01:40:36,Exception IndexError list index out of range in file /apis/system/ntp.py at line 347,test_ft_ntp_disable_enable_with_message_log,system/test_ntp.py,V3,
80,gw5,Regression,ft_ntp_existsconfig,Pass,Mapped,2023-12-25 01:40:49,Test case passed,test_ntp_exists_config,system/test_ntp.py,V3,
81,gw4,--NE--,auth_shared_keys ,--NE--,Mapped,2022-01-03 14:20:27,--NE-- Check if gw4 is Dead,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,,
82,gw4,--NE--,executing_server_key_in_appropriate_mode,--NE--,Mapped,2022-01-03 14:20:27,--NE-- Check if gw4 is Dead,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,,
83,gw4,--NE--,ft_tacacs_default_login,--NE--,Mapped,2022-01-03 14:20:27,--NE-- Check if gw4 is Dead,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,,
84,gw4,--NE--,ft_tacacs_single_server_for_auth,--NE--,Mapped,2022-01-03 14:20:27,--NE-- Check if gw4 is Dead,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,,
85,gw4,--NE--,ft_tacacs_ssh_login,--NE--,Mapped,2022-01-03 14:20:27,--NE-- Check if gw4 is Dead,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,,
86,gw4,--NE--,ft_tacacsauth_enable_pamfile_created,--NE--,Mapped,2022-01-03 14:20:27,--NE-- Check if gw4 is Dead,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,,
